packages/engine/Source/Core/ITwin.js

```diff
@@ -0,0 +1,306 @@
+ * @property {boolean} includeLines
+ * @property {number} chordTol
+ * @property {number} angleTol
+ * @property {number} decimationTol
+ * @property {number} maxEdgeLength
+ * @property {number} minBRepFeatureSize
+ * @property {number} minLineStyleComponentSize
+ */
+
+/**
+ * @typedef {Object} ViewDefinitionFilter
+ * @property {string[]} models Array of included model IDs.
+ * @property {string[]} categories Array of included category IDs.
+ * @property {string[]} neverDrawn Array of element IDs to filter out.
+ */
+
+/**
+ * @typedef {Object} StartExport
+ * @property {string} iModelId
+ * @property {string} changesetId
+ * @property {ExportType} exportType Type of mesh to create. Currently, only GLTF and 3DFT are supported and undocumented CESIUM option
+ * @property {GeometryOptions} geometryOptions
+ * @property {ViewDefinitionFilter} viewDefinitionFilter
+ */
+
+/**
+ * @typedef {Object} Link
+ * @property {string} href
+ */
+
+/**
+ * @typedef {Object} Export
+ * @property {string} id
+ * @property {string} displayName
+ * @property {ExportStatus} status
+ * @property {StartExport} request
+ * @property {{mesh: Link}} _links
+ */
+
+/**
+ * @typedef {Object} ExportResponse
+ * @property {Export} export
+ */
+
+/**
+ * Default settings for accessing the iTwin platform.
+ *
+ * Keys can be created using the iModels share routes {@link https://developer.bentley.com/apis/imodels-v2/operations/create-imodel-share/}
+ *
+ * An ion access token is only required if you are using any ion related APIs.
+ * A default access token is provided for evaluation purposes only.
+ * Sign up for a free ion account and get your own access token at {@link https://cesium.com}
+ *
+ * @see IonResource
+ * @see IonImageryProvider
+ * @see IonGeocoderService
+ * @see createWorldImagery
+ * @see createWorldTerrain
+ * @namespace ITwin
+ */
+const ITwin = {};
+
+/**
+ * Gets or sets the default iTwin access token.
+ *
+ * TODO: I'm not sure we can even do this kind of access token. Each route seems to need it's own scopes
+ * and we may not be able to guarantee this "top level token" has them all
+ * So far we use
+ * `mesh-export:read` for loading meshes GET /mesh-export(s)
+ * `mesh-export:modify` if we want to include a function to create an export
+ * `itwin-platform` if we want to use the iModel shares ourselves  GET /imodels/{id}/shares
+ *
+ *
+ * @type {string|undefined}
+ */
+ITwin.defaultAccessToken = undefined;
+
+/**
+ * Gets or sets the default iTwin API endpoint.
+ *
+ * @type {string|Resource}
+ * @default https://api.bentley.com
+ */
+ITwin.apiEndpoint = new Resource({
+  url: "https://api.bentley.com",
+});
+
+/**
+ * @param {string} exportId
+ */
+ITwin.getExport = async function (exportId) {
+  if (!defined(ITwin.defaultAccessToken)) {
+    throw new DeveloperError("Must set ITwin.defaultAccessToken first");
+  }
+
+  const headers = {
+    Authorization: ITwin.defaultAccessToken,
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
```

Comment:
This is fine for this specific use case where youre using just a single API (mesh-export), but might prove to be an issue when you are mixing APIs(iTwins, iModels, etc)
We use request headers to denote which verison of an API to use. So far, most APIs are still on v1, but there are some for example the iModels APIs which are on v2, and the the `accept` header would be need to be `application/vnd.bentley.itwin-platform.v2+json`

Please see [API Version Policy](https://developer.bentley.com/apis/overview/lifecycle/#api-versions)

Reply:
I was going off the sample code provided to us and the [documentation](https://developer.bentley.com/apis/mesh-export/operations/get-export/#request-headers) which states for every route in the mesh export API that:

> Setting to `application/vnd.bentley.itwin-platform.v1+json` is recommended

If the docs are wrong then they should be updated. On top of that I'm currently setting this per request so different routes could definitely use different values if needed for different versions as you say. As a consumer of the API I would fully expect all the routes on the same API to still work with each other if versioning is done this way.

Reply:
Yes this will work, the entire mesh-export service apis are still on v1 so the accept header here is right.

My apologies, I jumped the gun when I had initially commented this thinking you were going to use multiple api routes in a single function.